src/scripts/main.js

```diff
@@ -18,10 +18,29 @@ const food = {
   },
 };
 
+const newTree = document.createElement('div');
+
+newTree.setAttribute('id', 'tree');
+
+document.body.prepend(newTree);
+
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  element.append(list);
+
+  for (const key in data) {
```

Comment:
Using a `for...in` loop is appropriate here since you are iterating over the keys of an object. However, if you need to filter out inherited properties, consider using `Object.hasOwnProperty()`.